config/app.go

```diff
@@ -16,6 +17,56 @@ type ValidationError struct {
 	EnvVar []string
 }
 
+// Github Pull Request type.
+type PullRequest struct {
+	Host   string
+	Owner  string
+	Repo   string
+	Number int
+}
+
+// LoadFromURL parses Github Pull Request URL
+// into PullRequest object.
+func (p *PullRequest) LoadFromURL() error {
```

Comment:
since you start with empty object anyway `pr := PullRequest{}`, would be better to have PullRequest as return type. and for testing use the required string as input.
```suggestion
func LoadFromURL(pullRequestUrl string) (*PullRequest, error) {
```